interview-questions.md

```diff
@@ -5,55 +5,55 @@ First, without external resources. Challenge yourself to answer from memory.
 
 Then, research the question to expand on your answer. Even if you feel you have answered the question completely on your own, there is always something more to learn. Write your researched answer in your OWN WORDS.
 
-1. What is a block in Ruby?
+1. What is a block in Ruby? 
 
-  Your answer:
+  Your answer: A block in Ruby is a packet of information inside a component nesting in something
 
-  Researched answer:
+  Researched answer: The code that is passed to a method is called a block. Anonymous functions are also a block
 
 
 
 2. What is a gem?
 
-  Your answer:
+  Your answer:A jem is also a packet of information, but a series of specific packets that presume a function. 
```

Comment:
There are a lot of new vocab words coming at you. But it is important to double check your spelling when submitting documentation.